packages/eslint-config-airbnb-base/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Airbnb's base JS ESLint config, following our styleguide",
   "main": "index.js",
   "scripts": {
+    "prelint": "editorconfig-tools check *.js rules/*.js test/*.js",
```

Comment:
huh, i wasn't aware you could "pre" any old script. Seems this works all the way back to npm 1.